### PR TITLE
[Task 1847320]1. add data modules to mysql-lib

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/resources/com.microsoft.azure.toolkit.operation/titles_mysql.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+mysql|server.create=create MySQL server({})
+mysql|server.start=start MySQL server({})
+mysql|server.stop=stop MySQL server({})
+mysql|server.restart=restart MySQL server({})
+mysql|server.delete=delete MySQL server({})
```
Comment (shenqianjin) [inline azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/resources/com.microsoft.azure.toolkit.operation/titles_mysql.properties:5]: MySQL -> Azure Database for MySQL server

Comment (andxu) [reply azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/resources/com.microsoft.azure.toolkit.operation/titles_mysql.properties:5]: No need, 
image

File: azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/service/AbstractMySqlFirewallRuleCreator.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import com.microsoft.azure.toolkit.lib.common.task.ICommittable;
+import lombok.Getter;
+
+@Getter
+public abstract class AbstractMySqlFirewallRuleCreator implements ICommittable<MySqlFirewallRule> {
+    private String name;
+    private String startIpAddress;
+    private String endIpAddress;
+
+    public AbstractMySqlFirewallRuleCreator withName(String name) {
```
Comment (Flanker32) [inline azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/service/AbstractMySqlFirewallRuleCreator.java:16]: Do we need a creator for firewall rules? IMO, firewall rules are more likely the attributes of mysql, like the app settings for app service

Comment (shenqianjin) [reply azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/service/AbstractMySqlFirewallRuleCreator.java:16]: I prefer to do it like this code.

Comment (andxu) [reply azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/service/AbstractMySqlFirewallRuleCreator.java:16]: In the future, we may need to use this logic to  create firewall
see `az mysql server firewall-rule create`

**Review by shenqianjin — APPROVED** [review]
LGTM now

File: azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/service/AbstractMySqlFirewallRuleCreator.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        this.name = name;
+        return this;
+    }
+
+    public AbstractMySqlFirewallRuleCreator wihStartIpAddress(String startIpAddress) {
```
Comment (wangmingliang-ms) [inline azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/service/AbstractMySqlFirewallRuleCreator.java:21]: typo

File: azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/resources/com/microsoft/azure/toolkit/operation/titles_mysql.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+mysql|server.create=create MySQL server({})
```
Comment (wangmingliang-ms) [inline azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/resources/com/microsoft/azure/toolkit/operation/titles_mysql.properties:1]: `create MySQL server({0})`

**Pull request merged**